public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php

```diff
@@ -412,7 +412,11 @@ public function metabox_save( $post_id, $post ) {
 			return;
 		}
 
-		// Make sure the requset came from the edit post screen.
+		if ( $this->get_event_type() !== get_post_type() ) {
```

Comment:
Using `$post` might be a bit safer than `get_post_type()`. I can't think of a case where they'd be different, but there might be some.